docs/spec/spec.md

```diff
@@ -182,6 +183,87 @@ type = "stderr"
 path = "./logs/app.log"
 ```
 
+### 3.5. Configure log rotation
+
+Log rotation helps manage log file sizes by automatically creating backup files when certain conditions are met. This prevents log files from growing indefinitely and consuming excessive disk space.
+
+Log rotation is optional and can be configured for file destinations. If no rotation configuration is provided, logs are written without rotation. When configured, a rotation policy defines when files should be rotated. The following rotation policies are available:
+
+```ballerina
+public enum RotationPolicy {
```

Comment:
Need to update the `FileOutputDestination` definition in the spec with the rotation policy field